test/common/packagelib.py

```diff
@@ -87,6 +88,13 @@ def setUp(self):
             self.machine.execute(f"mkdir -p {empty_repo_dir} || true")
             self.machine.execute(f"repo-add {empty_repo_dir}/empty.db.tar.gz")
             self.machine.write("/etc/pacman.conf", config)
+            # Clean up possible leftover lockfile
+            self.machine.execute("""
+            if [ -f /var/lib/pacman/db.lck ]; then
+                fuser -k /var/lib/pacman/db.lck && sleep 0.2 || true;
+                rm -f /var/lib/pacman/db.lck || true;
```

Comment:
Please don't `|| true` this. At this point we already know the file exists, so I'd even argue for dropping the `-f`.